agent/agentssh/agentssh.go

```diff
@@ -272,6 +279,18 @@ func (s *Server) sessionHandler(session ssh.Session) {
 		extraEnv = append(extraEnv, fmt.Sprintf("DISPLAY=:%d.0", x11.ScreenNumber))
 	}
 
+	if s.fileTransferBlocked(session) {
+		s.logger.Warn(ctx, "file transfer blocked", slog.F("session_subsystem", session.Subsystem()), slog.F("raw_command", session.RawCommand()))
```

Comment:
out of scope: we may want to eventually have a way to surface this to admins as they may be interested in this sort of thing.

Reply:
a new metric?